src/plasma_manager.c

```diff
@@ -103,6 +79,9 @@ struct client_object_connection {
   char **manager_vector;
   /** The number of manager locations in the array manager_vector. */
   int manager_count;
+  /** The index of the next manager in manager_vector that we should try to
+   * contact. */
+  int next_manager;
```

Comment:
Ok, after looking through the rest of the PR, is it correct to say that there will always be another manager to try because we will just cycle through the managers repeatedly?